Application/Telegram/MainActions/NewBookingCommand.cs

```diff
@@ -41,11 +44,47 @@ public static async Task HandleAsync(Update update, TelegramBotClient telegraBot
                 case UserState.NewBookingIsSelectedSelectingEndDateMonth:
                     await new SendDayCommand(mediator, telegraBotClient).SendAsync(update.CallbackQuery, "Please select end date day", "BACKNewBookingIsSelected");
                     DateHelper.EndMonthUpdater(update.CallbackQuery, ref user);
+                    UserStateStorage.UserStateUpdate(update.CallbackQuery.From.Id, UserState.NewBookingIsSelectedSelectingPcOption);
+                    return;
+
+                case UserState.NewBookingIsSelectedSelectingPcOption:
+                    DateHelper.EndDayUpdater(update.CallbackQuery, ref user);
+                    await new ProvideButtons(telegraBotClient).SendAsync(update.CallbackQuery,new List<string> { "Yes", "No", "BACK" },  "Would you like workplace with PC", 2, "NewBookingIsSelected");
+                    UserStateStorage.UserStateUpdate(update.CallbackQuery.From.Id, UserState.NewBookingIsSelectedSelectingMonitorOption);
+                    return;
+
+                case UserState.NewBookingIsSelectedSelectingMonitorOption:
+                    UserStateStorage.userInfo[update.CallbackQuery.From.Id].HasPc = update.CallbackQuery.Data == "Yes" ? true : false;
```

Comment:
please make an empty space between new commands (separate line 57 and 58 by new line)